docs/shadow-vs-active-enforcement/README.mdx

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 6
+title: 'Shadow vs active enforcement'
+---
+
+When installing Otterize, the intents operator can be [configured](/reference/configuration/intents-operator/helm-chart) to enforce access controls or not.
+
+By default, the intents operator enforces access controls by creating, updating and deleting network policies and Kafka ACLs, for example, but all enforcement can be disabled by setting `enableEnforcement` to be `false`, or more granularly setting `enableNetworkPolicyCreation` and/or `enableNetworkPolicyCreation` to be `false`.
+
+## Active enforcement
+
+When enforcement is active &mdash; when it is not disabled &mdash; the intents operator processes applied client intents files and turns each into access authorizations from the client to its called servers: if network policies are active (and supported by the CNI in the Kubernetes cluster), that ensuring there are network policies from that client to all its called servers, and removing any other previous network policies from that client; and similarly for Kafka ACLs to Kafka servers.
+
+In general, of course, this will immediately block access from services that have not declared their intents to call the now-protected servers. That is, after all, why you use intent-based access control (IBAC).
+
+Whether this is a good thing or a problem depends on your situation. There are multiple ways of rolling out IBAC across the many services that may be running in some environment or Kubernetes cluster.
+- For example, you can insist that all services are protected by default, with a global default-deny network policy, and in the case of Kafka with an appropriate restrictive access policy (see the [Otterize KafkaServiceConfig documentation](/reference/configuration/intents-operator/configuration)). All clients must then declare and apply intents in order to reach their intended servers.
+- You can bootstrap these intents by generating the client intents files using the Otterize network mapper (see [this quick visual tutorial](/quick-visual-tutorials/visual-ibac-network-policies#protect-everything-easily)), and then hand them over to the client developer teams to own and update as their code evolves.
+- Or you can gradually roll out IBAC, starting with just a few services and expanding to more (see [this network policies tutorial](/quick-visual-tutorials/visual-ibac-network-policies) or [this Kafka ACLs tutorial](/quick-visual-tutorials/visual-ibac-kafka-k8s)).
+
+To help you manage the rollout of IBAC, consider starting with shadow enforcement.
+
+## Shadow enforcement
+
+When enforcement is not active &mdash; when it is disabled &mdash; the intents operator still processes client intents files, but it does not then turn them into access authorizations: no network policies are created or deleted or changed in any way, and similarly no Kafka ACLs are created, deleted, or modified.
```

Comment:
This version, without "when it is disabled", was easier for me to parse, the negation on "not active" followed by "disabled" confused me for a moment.
```suggestion
When enforcement is not active, the intents operator still processes client intents files, but it does not then turn them into access authorizations: no network policies are created or deleted or changed in any way, and similarly no Kafka ACLs are created, deleted, or modified.
```